.github/workflows/release.yml

```diff
@@ -62,6 +68,8 @@ jobs:
           IMAGE_REGISTRY_USER_NAME: eclipse
         run: |
           CHE_VERSION=${{ github.event.inputs.version }}
+          DWO_VERSION=${{ github.event.inputs.dwoVersion }}
```

Comment:
in case someone forgets the "v" prefix, it would be good to check for it and add it if it's missing.